internal/config.go

```diff
@@ -44,3 +48,60 @@ func UnmarshallBuffer(c CryptoHandler, encContent []byte, decrypt bool) (ConfigS
 	}
 	return config, nil
 }
+
+type ConfigFileReq struct {
+	Name        string   `json:"name"`
+	Value       string   `json:"value"`
+	Unencrypted bool     `json:"unencrypted"`
+	OnChanged   []string `json:"on-changed,omitempty"`
+}
+
+type ConfigCreateRequest struct {
+	Reason string          `json:"reason"`
+	Files  []ConfigFileReq `json:"files"`
+}
+
+func updateConfig(app *App, client *http.Client, pubkey string) error {
```

Comment:
As I understand now this function is unused without the ifdef, making the linter unhappy.

This change is extremely trivial though; so I recommend to take it wherever you planned to reuse this function (as a separate PR commit).

Reply:
I'm trying a slightly different approach. I've changed our linter to include vpn.go which found a bug.

I think doing it all here is going to make your life easier when you finally see the cert-rotation PR

Reply:
Take a look at my force push and make sure you like it.